usecases/gitops.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: GitOps
+description: This document describes how Flipt can be used with GitOps workflows
+mode: wide
+---
+
+## Overview
+
+GitOps is a way to do Continuous Delivery, it works by using Git as a single source of truth for declarative infrastructure and applications. With Git at the center of your delivery pipelines, developers can make pull requests to accelerate and simplify application deployments.
+
+Flipt is uniquely suited to work with GitOps workflows because of our declarative backends and ability to be configured completely via YAML.
+
+We believe that feature flags are a form of configuration and should be treated as such. This means that feature flags should be able to be stored in the same repository as the code that uses them. This allows developers to make changes to both the code and the feature flags in the same pull request. This also allows developers to use the same GitOps tooling to deploy both their code and their feature flags to production.
+
+## Git Backend
+
+Flipt's declarative storage backends allow it to run without a database at all. Flipt can be configured to load its feature flag data in the following ways:
+
+- From a file on the local filesystem
+- From a remote **git repository**
+- From an OCI compliant container image registry
+- From from any of the 3 major cloud computing object storage services (AWS, Google Cloud, and Azure)
+
+Our git support enables you to evaluate feature flags across different branches and tags. This allows developers to test feature flags in a staging environment before merging them into production, or leverage preview environments to test feature flags in isolation.
+
+Read our [Get Going with GitOps](https://www.flipt.io/docs/guides/get-going-with-gitops) guide to learn how to get started with GitOps and Flipt.
+
+## CI/CD Integration
+
+Our [GitHub Action](/tooling/github-actions) allows you to easily integrate Flipt into your CI/CD pipelines. This allows you to install Flipt into your CI pipeline and run tests against your feature flags before deploying your code to production.
+
+You can also import your feature flag data from your repository, or any other source, into Flipt using our [Import Command](/cli/commands/import). This allows you to import your feature flag data into Flipt before running your tests.
```

Comment:
I think it is better to encourage folks to use `local` backend here for CI. As it avoid the need to run Flipt with a DB and have a separate `import` step. You just run Flipt with `local` and point it at the relevant directory.

Reply:
why not both? updated